### PR TITLE
Created daytime_contacts model

### PR DESCRIPTION
**Pull request by Het7**
## Notion ticket link

[Create daytime_contacts model](https://www.notion.so/uwblueprintexecs/Create-daytime_contacts-model-b49208767f4d4559b5da583f9821a6e5)

## Implementation description
* Created a daytime_contacts model based on the schema provided in the ticket

## Checklist
- [x] My PR name is descriptive and in imperative tense
- [x] My commit messages are descriptive and in imperative tense. My commits are atomic and trivial commits are squashed or fixup'd into non-trivial commits
- [x] I have run the appropriate linter(s)
- [x] I have requested a review from the PL, as well as other devs who have background knowledge on this PR or who will be building on top of this PR

### CODE REVIEW

File: backend/python/app/models/daytime_contacts.py [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+    id = db.Column(db.Integer, primary_key=True, nullable=False)
+    contact_first_name = db.Column(db.String, nullable=False)
+    contact_last_name = db.Column(db.String, nullable=False)
+    address_id = db.Column(db.Integer, db.ForeignKey("addresses.id"), nullable=False)
+    address = db.relationship("Address")
+    phone_number = db.Column(db.String, nullable=False)
+    type = db.Column(type_enum)
+
+    def to_dict(self, include_relationships=False):
+        # define the entities table
```
Comment (jennchenn) [inline backend/python/app/models/daytime_contacts.py:21]: nit: remove these comments

**Review by jennchenn — CHANGES_REQUESTED** [review]
i don't see your migration here! do you mind generating it first and then re-requesting review?

**Comment by Het7** [comment]
> i don't see your migration here! do you mind generating it first and then re-requesting review?

Done!

File: backend/python/migrations/versions/4dfecb9b3a37_daytime_contacts.py [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+        ),
+        sa.PrimaryKeyConstraint("id"),
+    )
+    # ### end Alembic commands ###
+
+
+def downgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.drop_table("daytime_contacts")
+    # ### end Alembic commands ###
```
Comment (jennchenn) [inline backend/python/migrations/versions/4dfecb9b3a37_daytime_contacts.py:44]: add:
`op.execute("DROP TYPE daytime_contacts_type;")`

File: backend/python/migrations/versions/4dfecb9b3a37_daytime_contacts.py [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+    op.create_table(
+        "daytime_contacts",
+        sa.Column("id", sa.Integer(), nullable=False),
+        sa.Column("contact_first_name", sa.String(), nullable=False),
+        sa.Column("contact_last_name", sa.String(), nullable=False),
+        sa.Column("address_id", sa.Integer(), nullable=False),
+        sa.Column("phone_number", sa.String(), nullable=False),
+        sa.Column(
+            "type",
+            sa.Enum("School", "Daycare", name="daytime_contact_enum"),
```
Comment (jennchenn) [inline backend/python/migrations/versions/4dfecb9b3a37_daytime_contacts.py:29]: nit: rename
```suggestion
            sa.Enum("School", "Daycare", name="daytime_contacts_type"),
```

**Review by jennchenn — CHANGES_REQUESTED** [review]
some minor changes - please check that you can run `flask db upgrade && flask db downgrade && flask db upgrade` without issue once you make the changes!

**Comment by Het7** [comment]
> some minor changes - please check that you can run `flask db upgrade && flask db downgrade && flask db upgrade` without issue once you make the changes!

Yup I'm able to run it without any issues!

File: backend/python/app/models/daytime_contacts.py [hunk 4]
```diff
@@ -0,0 +1,32 @@
+from sqlalchemy import inspect
+from sqlalchemy.orm.properties import ColumnProperty
+
+from . import db
+
+type_enum = db.Enum("School", "Daycare", name="daytime_contact_enum")
+
+
+class daytime_contacts(db.Model):
```
Comment (jennchenn) [inline backend/python/app/models/daytime_contacts.py:9]:
```suggestion
class DaytimeContact(db.Model):
```

**Review by jennchenn — APPROVED** [review]
:shipit:

**Pull request merged**